apps/console/src/features/applications/components/wizard/minimal-application-create-wizard.tsx

```diff
@@ -456,27 +456,40 @@ export const MinimalAppCreateWizard: FunctionComponent<MinimalApplicationCreateW
+                                                                        ...getTechnologyLogos()
+                                                                    }[ subTemplate.image ]
+                                                                }
+                                                                size="x120"
+                                                                className="sub-template-selection-card"
+                                                                header={ subTemplate.name }
+                                                                selected={ selectedTemplate.id === subTemplate.id }
+                                                                onClick={ () => {
+                                                                    if (!subTemplate.previewOnly) {
+                                                                        setSelectedTemplate(subTemplate);
+                                                                        loadTemplateDetails(subTemplate.id, subTemplate);
+                                                                    }
+                                                                } }
+                                                                imageSize="mini"
+                                                                contentTopBorder={ false }
+                                                                showTooltips={ !subTemplate.previewOnly }
+                                                                disabled={ subTemplate.previewOnly }
+                                                                data-testid={ `${ testId }-${ subTemplate.id }-card` }
+                                                            />
                                                         }
-                                                        size="x120"
-                                                        className="sub-template-selection-card"
-                                                        header={ subTemplate.name }
-                                                        selected={ selectedTemplate.id === subTemplate.id }
-                                                        onClick={ () => {
-                                                            setSelectedTemplate(subTemplate);
-                                                            loadTemplateDetails(subTemplate.id, subTemplate);
-                                                        } }
-                                                        imageSize="mini"
-                                                        contentTopBorder={ false }
-                                                        showTooltips={ true }
-                                                        data-testid={ `${ testId }-${ subTemplate.id }-card` }
+                                                        content={ "Coming soon" }
```

Comment:
Shall we refer this from `common` i18n namespace?

Reply:
Addressed with common i18n namespace